contracts/CentralizedBlockRelay.sol

```diff
@@ -0,0 +1,196 @@
+pragma solidity ^0.5.0;
+
+import "./BlockRelayInterface.sol";
+
+/**
+ * @title Block relay contract
+ * @notice Contract to store/read block headers from the Witnet network
+ * @author Witnet Foundation
+ */
+
+
+contract CentralizedBlockRelay  is BlockRelayInterface {
+
+  struct MerkleRoots {
+    // hash of the merkle root of the DRs in Witnet
+    uint256 drHashMerkleRoot;
+    // hash of the merkle root of the tallies in Witnet
+    uint256 tallyHashMerkleRoot;
+  }
+  struct Beacon {
+    // hash of the last block
+    uint256 blockHash;
+    // epoch of the last block
+    uint256 epoch;
+  }
+
+  // Address of the block pusher
+  address witnet;
+  // Last block reported
+  Beacon public lastBlock;
+
+  mapping (uint256 => MerkleRoots) public blocks;
+
+  // Event emitted when a new block is posted to the contract
+  event NewBlock(address indexed _from, uint256 _id);
+
+  constructor() public{
```

Comment:
I normally expect the constructor after the modifiers, right before the first method.